apps/api/src/db/schema.dbml

```diff
@@ -93,7 +93,7 @@ table application_contents {
   institution_building varchar(255)
   project_title text
   project_website text
-  project_abstract text
+  project_background text
   project_methodology text
   project_summary text
   project_publication_urls text[]
```

Comment:
We have a dbml getting updated here but it's diverging from `docs/model/data-model.dbml`, don't know if there's benefit to keeping both